src/model/Model_Account.cpp

```diff
@@ -30,7 +30,8 @@ const std::vector<std::pair<Model_Account::TYPE, wxString> > Model_Account::TYPE
     std::make_pair(Model_Account::CHECKING, wxTRANSLATE(wxString("Checking"))),
     std::make_pair(Model_Account::TERM, wxTRANSLATE(wxString("Term"))),
     std::make_pair(Model_Account::INVESTMENT, wxTRANSLATE(wxString("Investment"))),
-    std::make_pair(Model_Account::CREDIT_CARD, wxTRANSLATE(wxString("Credit Card")))
+    std::make_pair(Model_Account::CREDIT_CARD, wxTRANSLATE(wxString("Credit Card"))),
```

Comment:
May be 'wxTRANSLATE(wxString("Credit Card"))'
should be 
'wxString(wxTRANSLATE("Credit Card"))'.
'wxTRANSLATE' needed only if '_("Credit Card")' not exists in the projuct.